### ISSUE TITLE
Add replica count for dapr status -k command

### ISSUE THREAD

**Issue by yaron2** [dapr/cli#410]
When running `dapr status -k`, the output will contain all the Dapr control plane pods, their version and status, but without a replica count.

As Dapr will have a HA deployment mode, it's important to be able to inform the user how many replicas are running.

Current output:

```
  NAME                   NAMESPACE    HEALTHY  STATUS   VERSION  AGE  CREATED
  dapr-placement         dapr-system  True     Running  0.8.0    1m   2020-07-20 10:00.17
  dapr-sentry            dapr-system  True     Running  0.8.0    1m   2020-07-20 10:00.17
  dapr-operator          dapr-system  True     Running  0.8.0    1m   2020-07-20 10:00.17
  dapr-sidecar-injector  dapr-system  True     Running  0.8.0    1m   2020-07-20 10:00.17
```

Expected output:

```
  NAME                   NAMESPACE    HEALTHY  STATUS   VERSION REPLICAS  AGE  CREATED
  dapr-placement         dapr-system  True     Running  0.8.0   1         1m   2020-07-20 10:00.17
  dapr-sentry            dapr-system  True     Running  0.8.0   1         1m   2020-07-20 10:00.17
  dapr-operator          dapr-system  True     Running  0.8.0   1         1m   2020-07-20 10:00.17
  dapr-sidecar-injector  dapr-system  True     Running  0.8.0   1         1m   2020-07-20 10:00.17
```

**Comment by aszego** [comment]
Hasn't this been solved with #456 ?

**Comment by yaron2** [comment]
That is correct, thank you.
